README.md

```diff
@@ -122,7 +122,7 @@ Also, please subscribe to the [Phaser World](https://phaser.io/community/newslet
 
 ### Source Code Examples
 
-During our development of Phaser 3 we created hundreds of examples, with the full source code and assets available. Until those are fully integrated with the Phaser web site, you'll have to browse them in the [Phaser 3 Labs](http://labs.phaser.io), or clone the [examples repo][examples]. Note: Not all examples work, sorry! We're tidying them up as fast as we can.
+During our development of Phaser 3 we created hundreds of examples using the full source code and assets. Until these examples are fully integrated into the Phaser website, you can browse them on [Phaser 3 Labs](http://labs.phaser.io), or clone the [examples repo][examples]. Note: Not all examples work, sorry! We're tidying them up as fast as we can.
```

Comment:
"using the full source code" doesn't make any sense in this context. Please revert this line to the previous version, which is saying that when we created the examples we also released the full code to them and their assets for devs to learn from.

Reply:
@photonstorm thanks for your feedback. I've committed this change.